operator/Makefile

```diff
@@ -152,10 +158,14 @@ manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) rbac:roleName=lokistack-manager crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
 .PHONY: test
-test: deps generate go-generate lint lint-prometheus manifests ## Run tests
+test: deps deps-api generate go-generate lint lint-prometheus manifests test-unit-api ## Run tests
 test: $(GO_FILES)
 	go test ./... -coverprofile cover.out
 
+.PHONY: test-unit-api
+test-unit-api: $(GO_FILES)
+	@cd ./apis/loki/ && go test ./... -coverprofile cover.out
```

Comment:
Isn't `cover.out` overwritten once the main test target finished?

Reply:
There will be two `cover.out` files. One in the api module and one in the operator module.

I wondered about this file as well ... do we use it somewhere? CI?